documentation/supported_plugins.md

```diff
@@ -209,3 +210,152 @@ config:
       parameters:
         key: ssh_password
 ```
+
+### `puppet_connect_data`
```

Comment:
@hestonhoffman This plugin's meant to be used by Puppet Connect users. Although it is a supported plugin that we (Skeletor) do want to ship with Bolt, I'm not sure if we should document this here, in Puppet Connect-specific docs, or both. I've been told you're a good person to ask about this, so am welcoming any ideas/suggestions here.

Reply:
Do we expect Bolt users who aren't also Connect users to use this plugin? If not, I don't think it should be included here and should just be in Connect docs with a note that the plugin can still be used with Bolt without breaking.

Reply:
I think this plugin will only be used by Connect users, so putting it in the Connect docs seems like the way to go. I'll remove them from this PR.

Should the plugin still have a release note in the commit?

Reply:
Probably not - you can do `!no-release-note` so it doesn't fail the check.

Reply:
Updated to remove the docs + add `!no-release-note`.